app/views/shared/_edit_mail_template.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for @mail_template do |f| %>
+    <%= f.label :body, 'Message:' %>
+    <%= f.text_area :body, :class => "wide_form", :cols => "10", :rows => "10"  %><br><br>
+
+<strong>Link to accept invitation:</strong>
```

Comment:
it's reasonable that the link_to element can't edit, but the regular text just above it?  I would have thought that perhaps that could be included in the top half element?